projects/plugins/jetpack/extensions/blocks/voice-to-content/edit.tsx

```diff
@@ -7,87 +7,214 @@ import {
 	playerPauseIcon,
 	useMediaRecording,
 } from '@automattic/jetpack-ai-client';
-import { useBlockProps } from '@wordpress/block-editor';
-import { Placeholder, Button } from '@wordpress/components';
+import { ThemeProvider } from '@automattic/jetpack-components';
+import { Button, Modal, Icon } from '@wordpress/components';
+import { useDispatch } from '@wordpress/data';
 import { useCallback } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
+import { external } from '@wordpress/icons';
+/**
+ * Internal dependencies
+ */
+import oscilloscope from './assets/oscilloscope.svg';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+function Oscilloscope( { audioURL } ) {
```

Comment:
Why are you passing audioURL here? Are you planning on using it later on?

Reply:
Yes, the idea is to make an actual oscilloscope later and remove the asset.